packages/core/src/middleware/koa-quota-guard.ts

```diff
@@ -13,7 +13,9 @@ export default function koaQuotaGuard<StateT, ContextT, ResponseBodyT>({
   quota,
 }: UsageGuardConfig): MiddlewareType<StateT, ContextT, ResponseBodyT> {
   return async (ctx, next) => {
-    await quota.guardKey(key);
+    if (ctx.method === 'POST' || ctx.method === 'PUT') {
```

Comment:
this will change all other routes. not sure if it's properiate

Reply:
Just checked, turned out we are only using the `koaQuotaGuard` on several POST requests. So I think it's OK

Reply:
even this i think there's still a better way or we can just keep the original implementation if we can achieve the goal for now

Reply:
Updated and make the `KoaQuotaGuard` configurable to http methods. Defaults to all.